docs/Installation-Guide.rst

```diff
@@ -570,11 +570,11 @@ The CUDA-based build is a separate implementation and requires an NVIDIA graphic
 Linux
 ^^^^^
 
-On Linux a CUDA version of LightGBM can be built using **CUDA**, **CMake** and **gcc**.
+On Linux a CUDA version of LightGBM can be built using **CUDA**, **CMake** and **gcc** or **Clang**.
 
 The following dependencies should be installed before compilation:
 
--  **CUDA** libraries. Please refer to `this detailed guide`_.
+-  **CUDA** libraries. Please refer to `this detailed guide`_. Pay great attention to the minimum required versions of host compilers listed in the table from that guide and use only recommended versions of compilers.
```

Comment:
I don't think we should re-type this table in our `CMakeLists.txt` for each supported CUDA version.

![image](https://user-images.githubusercontent.com/25141164/106395378-0f5f6b80-6413-11eb-89aa-83d5a01b178d.png)

Reply:
I agree! We're likely to make a mistake trying to replicate this all in CMakeLists.txt, and for the limited benefit of raising a more informative error message.